src/Components/Header.js

```diff
@@ -4,11 +4,12 @@ const Header = () => {
   return (
     <header>
       <div className="left">
-        <svg width="100" height="100">
-          <circle
+        <svg width="120" height="120">
```

Comment:
I have updated the logo image but you still haven't made it a link to the home page. I have marked that as incomplete on the board